jni/src/test/java/com/horizen/sigproofnative/NaiveThresholdSigProofTest.java

```diff
@@ -1,314 +1,314 @@
-            } else {
-                signatureList.add(new SchnorrSignature());
-            }
-        }
-
-        //Free memory from the secret keys
-        for (SchnorrKeyPair kp: keyPairList)
-            kp.getSecretKey().freeSecretKey();
-
-        createAndVerifyProof();
-    }
-
-    private void createAndVerifyProof() {
-
-        ClassLoader classLoader = getClass().getClassLoader();
-
-        String provingKeyPath = new File(classLoader.getResource("sample_params").getFile()).getAbsolutePath();
-        CreateProofResult proofResult = NaiveThresholdSigProof.createProof(btList, endEpochBlockHash, prevEndEpochBlockHash,
-                signatureList, publicKeyList, threshold, provingKeyPath);
-
-        assertNotNull("Proof creation must be successfull", proofResult);
-
-        String verificationKeyPath = new File(classLoader.getResource("sample_vk").getFile()).getAbsolutePath();
-
-        byte[] proof = proofResult.getProof();
-        long quality = proofResult.getQuality();
-
-        FieldElement constant = NaiveThresholdSigProof.getConstant(publicKeyList, threshold);
-        assertNotNull("Constant creation must be successfull", constant);
-
-        boolean isProofVerified = NaiveThresholdSigProof.verifyProof(btList, endEpochBlockHash,
-                prevEndEpochBlockHash, constant, quality, proof, verificationKeyPath);
-
-        assertTrue("Proof must be verified", isProofVerified);
-
-        quality = threshold - 1;
-        isProofVerified = NaiveThresholdSigProof.verifyProof(btList, endEpochBlockHash,
-                prevEndEpochBlockHash, constant, quality, proof, verificationKeyPath);
-
-        assertFalse("Proof must not be verified", isProofVerified);
-    }
-
-    @After
-    public void testFree(){
-        for (SchnorrPublicKey pk: publicKeyList)
-            pk.freePublicKey();
-        publicKeyList.clear();
-
-        for (SchnorrSignature sig: signatureList)
-            sig.freeSignature();
-        signatureList.clear();
-    }
-}
\ No newline at end of file
+//package com.horizen.sigproofnative;
+//
+//import com.horizen.librustsidechains.FieldElement;
+//import com.horizen.schnorrnative.SchnorrKeyPair;
+//import com.horizen.schnorrnative.SchnorrPublicKey;
+//import com.horizen.schnorrnative.SchnorrSecretKey;
+//import com.horizen.schnorrnative.SchnorrSignature;
+//import org.junit.Before;
+//import org.junit.After;
+//import org.junit.Test;
+//
+//import java.io.File;
+//import java.io.FileReader;
+//import java.util.ArrayList;
+//import java.util.List;
+//import java.util.Random;
+//
+//import static org.junit.Assert.assertNotNull;
+//import static org.junit.Assert.assertTrue;
+//import static org.junit.Assert.assertFalse;
+//
+//public class NaiveThresholdSigProofTest {
+//
+//    static int keyCount = 3;
+//    static long threshold = 2;
+//    static int backwardTransferCout = 10;
+//
+//    byte[] endEpochBlockHash = new byte[16];
```

Comment:
32 bytes

Reply:
With epoch 32 I am getting the error in UT: "Attempt to deserialize a field element over the modulus"

Reply:
Done